cassandra-4/src/main/java/io/debezium/connector/cassandra/Cassandra4CommitLogReadHandlerImpl.java

```diff
@@ -416,7 +520,7 @@ private void populatePartitionColumns(RowData after, PartitionUpdate pu) {
             try {
                 String name = columns.get(i).name.toString();
                 Object value = partitionKeys.get(columnMetadata.position());
-                CellData cellData = new CellData(name, value, null, CellData.ColumnType.PARTITION);
+                CellData cellData = new CellData(name, value, null, PARTITION);
```

Comment:
Would be good to move these (unrelated) cleanup change into separate commit. It makes the review more easy and also backports are much more easy to do when there's a need to backport a commit. Not anything which needs to be done now, just FIY - a suggestion for your next PRs to come.